manifests/g/Google/JapaneseIME/2.29.5370.0/Google.JapaneseIME.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Google.JapaneseIME
+PackageVersion: 2.29.5370.0
+InstallerLocale: ja-JP
+InstallerType: msi
+ProductCode: '{03E6F2DF-19B2-49B8-AEFB-E47FF09635B7}'
+ReleaseDate: 2024-02-21
+AppsAndFeaturesEntries:
+- DisplayName: Google 日本語入力
+  UpgradeCode: '{C1A818AF-6EC9-49EF-ADCF-35A40475D156}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.google.com/dl/release2/kjspmop3m4hu2sbbaotsynsgja_2.29.5370.0/GoogleJapaneseInput64-2.29.5370.0.msi
+  InstallerSha256: 6CA10E37F4792C91DEE6D334D57AFCD3233CE520F1E42D1B2342E5508F600474
```

Comment:
Adding https://dl.google.com/release2/ and https://edgedl.me.gvt1.com/edgedl/release2 will be better?
Old definition only have this `https://www.google.com/dl/release2`

Reply:
Unfortunately, I'm also unsure of which would be better. Feel free to submit an update PR later if it should be changed.